internal/git/github.go

```diff
@@ -31,12 +31,8 @@ type option func(*GithubRepository)
 func New(name string, options ...option) (*GithubRepository, error) {
 	r := GithubRepository{
 		Name:              name,
-		ServerURL:         "",
-		Organization:      "",
-		GithubAccessToken: "",
 		License:           "mit",
 		ProjectDir:        ".",
-		CloneURL:          "",
 	}
```

Comment:
This was another case where I was mid-thought, and had just not made it back to clean up.  Originally there was stuff there, and I blanked it, intending to remove fields that were still blank when I was finished.  I don't even remember what I was doing now.  I've slept since then.